src/Bridge/Symfony/Bundle/Test/Client.php

```diff
@@ -84,7 +84,7 @@ public function setDefaultOptions(array $defaultOptions): void
      *
      * @return Response
      */
-    public function request(string $method, string $url, array $options = []): ResponseInterface
+    public function request(string $method, string $url, array $options = [], array $files = []): ResponseInterface
```

Comment:
We can't change the method sign here as it should respect this interface: https://github.com/symfony/contracts/blob/master/HttpClient/HttpClientInterface.php#L84. Files aren't in the option list, we could maybe support this via the `extra` key?

Reply:
Updated to your suggestion